index.html

```diff
@@ -53,86 +61,153 @@ <h2>Multi-Post Stories</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
                 <ul class="langcontainer">
                     <li class="opaquebuttons">html</li>
                     <li class="opaquebuttons">bootstrap</li>
                     <li class="opaquebuttons">ruby</li>
                 </ul>
-                <input type="button" value="See Project" class="thiccbutton">
             </div>
-        </section>
-        <section class="myworks">
-            <div class="secondcard">
-                <h4>Professional Art <br> Printing Data</h4>
-                <p class="secondbio">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard.</p>
+        </div>
+        <div class="desktopcard2 grido">
+            <img alt="" src="imgs/deskcard2.svg">
+            <div class="cardinfo">
+                <h4>Data Dashboard Healthcare</h4>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
                 <ul class="langcontainer">
                     <li class="opaquebuttons">html</li>
                     <li class="opaquebuttons">bootstrap</li>
                     <li class="opaquebuttons">ruby</li>
                 </ul>
-                <input type="button" value="See Project" class="thiccbutton">
             </div>
-        </section>
-        <section class="myworks">
-            <div class="secondcard">
-                <h4>Professional Art <br> Printing Data</h4>
-                <p class="secondbio">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard.</p>
+        </div>
+        <div class="desktopcard2 grido">
+            <img alt="" src="imgs/deskcard3.svg">
+            <div class="cardinfo">
+                <h4>Data Dashboard Healthcare</h4>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
                 <ul class="langcontainer">
                     <li class="opaquebuttons">html</li>
                     <li class="opaquebuttons">bootstrap</li>
                     <li class="opaquebuttons">ruby</li>
                 </ul>
-                <input type="button" value="See Project" class="thiccbutton">
             </div>
-
-        </section>
-    </div>
+        </div>
+    </section>
     <section class="about">
         <div class="abouty">
-            <h1>About Me</h1>
-
+            <h1>About me</h1>
             <p class="descrip">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to"myForm me.</p>
             <input type="button" value="Get My Resume" class="flatbutton">
```

Comment:
- [ ] Your design is fantastic 👏. In order for your design to be perfect please fix the spacing in the buttons you should try to match the template as closely as possible which means following the guidelines for spacing around elements. 
## Your Design ( mobile-version)
![image](https://user-images.githubusercontent.com/18730106/144646520-3c57906d-02db-43a4-813b-716c93d644d4.png)
##  Figma Design (mobile-version)
![image](https://user-images.githubusercontent.com/18730106/144646590-d82c095d-7c3a-41b9-a95e-332c89649edc.png)
- [ ] Please 🙏also make the `button` not break on the desktop version like it is right now. ensure that the mobile version stays correct while modifying the desktop version.
## Your Design (desktop-version)
![image](https://user-images.githubusercontent.com/18730106/144647235-16863be1-fc4e-46f8-885c-23cbf0229a78.png)

## Figma Design (desktop-version)
![image](https://user-images.githubusercontent.com/18730106/144647294-ba98fbbe-c51e-40fb-845b-f9661e097879.png)